Source/Fuse.Reactive.Bindings/Instance.uno

```diff
@@ -145,10 +145,15 @@ namespace Fuse.Reactive
 			The `TemplateSource` property, along with the templates in the source, as well as the `TemplateKey`, must be set prior to
 			rooting to take effect.
 		*/
-		public Visual TemplateSource
+		public ITemplateSource TemplateSource
 		{
-			get; set;
+			get { return _weakTemplateSource; }
+			set { _weakTemplateSource = value; }
 		}
+		//https://github.com/fusetools/fuselibs-public/issues/135
+		[WeakReference]
+		ITemplateSource _weakTemplateSource;
```

Comment:
Well, since you wrote a nice comment I'll use it. :)

I guess if it's short it makes sense, but sometimes I prefer links since the issue can be long and hard to understand (I don't think pasting stories into the code makes sense).